ArkKit/ArkActionContext.swift

```diff
@@ -1,4 +1,8 @@
+/**
+ * A facade which encapsulates all the sub-contexts used in the execution of an action in `ArkBlueprint`.
+ */
 struct ArkActionContext {
     let ecs: ArkECS
-    let events: ArkEventContext
+    let events: ArkEventManager
```

Comment:
For `ecs: ` and `events: `, we can probably use the respective context protocols instead of the concrete implementations:

```swift
let ecs: ArkECSContext
let events: ArkEventContext
let display: ArkDisplayContext
```